pkg/vm/engine/tae/catalog/schema.go

```diff
@@ -1061,9 +1062,29 @@ func GetAttrIdx(attrs []string, name string) int {
 	panic("logic error")
 }
 
-type ComputeIndexInfo struct {
-	Name      string
-	TableName string
-	Unique    bool
-	Field     []string
+type UniqueIndexInfo struct {
```

Comment:
please consider storing bytes of these structs in schema, and deserializing them when fetching.